### DISCUSSION TITLE
[Converga.read and input format ][Hw6]

### DISCUSSION THREAD

**Discussion by antoineschutz** [prosyslab-classroom/cs348-information-security#748] [Q&A]
Name: Antoine Schute

Hello , I have a problem concerning the use of coverage.read

If I try to execute 
	`./fuzzer example01 string_seed out_dir`
In my main directory , I get "example01 not found".

So I modified the command to 
	`./fuzzer test/example01 test/string_seed out_dir`

Everything works well , until I try to call the Coverage.read function :

In my code I call the Coverage.read function with env.Coverage_file

`Fatal error: exception Sys_error("/home/student/…/test/example01.c.gcov: No such file or directory")`

I've to move every example* file in the main directory to see if it worked , but I think we're not allowed to change the directory structure.
how should i proceed ?

**Comment by m-spitfire** [comment]
1. Your command assumes wrong executable name. The name of exe is `example02` not `example1`.
2. You should execute fuzzer inside of `test/` directory with `../fuzzer example01 string_seed out_dir`. Please see the readme carefully on assignment repo.

**Comment by antoineschutz** [comment]
Oh indeed thank you !
